.github/workflows/review-release-notes.yml

```diff
@@ -1,32 +1,38 @@
 name: Review Release Notes
 on: pull_request
+
+env:
+  CHANGED_FILES_DELIMITER: ";"
+
 jobs:
   release_notes_review:
     runs-on: ubuntu-latest
     if: github.repository == 'demisto/content'
     steps:
+
       - name: Checkout
         uses: actions/checkout@v3
         with:
           fetch-depth: 2
+
       - name: Get changed files
         id: changed-files
-        uses: tj-actions/changed-files@v18.7
+        uses: tj-actions/changed-files@v35.1.0
         with:
-          separator: ';'
-      - name: Setup Python
-        uses: actions/setup-python@v3
-        with:
-          python-version: '3.9'
-      - name: Setup Poetry
-        uses: Gr1N/setup-poetry@v7
-      - name: Install Python Dependencies
+          separator: ${{ env.CHANGED_FILES_DELIMITER }}
+          files: |
+            Packs/**/ReleaseNotes/**
+          since_last_remote_commit: false
+
+      - name: Install SDK
+        if: ${{ steps.changed-files.outputs.all_changed_files }}
```

Comment:
what doe4s this condition means?

Reply:
This condition is added to two steps (_Install SDK_ and _Review Release Notes_). We're checking whether there are any outputs from the _Get Changed Files_ (which should only include Release Notes with this change) step. 

If not, we skip both _Install SDK_ and _Review Release Notes_ since there's no need to run them if there are no Release Notes to review. 

The elegant solution would be to terminate the job successfully in _Install SDK_ step and not have to have the same condition twice but apparently [there's no built-in way to do so in GitHub Actions yet](https://github.com/actions/runner/issues/662).

See [here](https://github.com/tj-actions/changed-files#outputs) for more info about the `changed-files` output used in the conditional.

Reply:
@kgal-pan 
>(which should only include Release Notes with this change)

Not sure I understand why it will include only release notes files, according to docs:
>Returns all changed files i.e. a
combination of all added, copied, modified
and renamed files (ACMR)

Reply:
It will only include release notes because of the [glob pattern supplied to the `files` key](https://github.com/demisto/content/blob/CIAC-4970-run-release-notes-check-only-if-xsoar-supported-pa/.github/workflows/review-release-notes.yml#L23-L24).

Reply:
great

Reply:
great